tests_e2e/tests/publish_hostname/publish_hostname.py

```diff
@@ -65,7 +65,7 @@ def check_and_install_dns_tools(self):
         except CommandError as e:
             if "dig: command not found" in e.stderr:
                 distro = self._ssh_client.run_command("get_distro.py").rstrip().lower()
-                if "debian_9" in distro:
+                if "debian_9" in distro or "debian_10" in distro:
```

Comment:
why was it that we use in instead of ==?

Reply:
the get_distro.py script returns a string which includes the minor version of the distro too

distro is something like debian_1013, so the test is using in since is it only cares about the major version

Reply:
ok... should we be using startswith instead?

Reply:
Updated